Libraries/oneMKL/fourier_correlation/README.md

```diff
@@ -1,34 +1,34 @@
 # Fourier Correlation Sample
-Fourier correlation has many applications, e.g.: measuring the similarity of two 1D signals, finding the best translation to overlay similar images, volumetric medical image segmentation, etc. This sample shows how to implement 1D and 2D Fourier correlation using SYCL, oneMKL, and oneDPL kernel functions.
+The cross-correlation has many applications, *e.g.*, measuring the similarity of two one-dimensional signals, finding the best translation to overlay similar images, volumetric medical image segmentation, etc. This sample shows how to implement one-dimensional and two-dimensional cross-correlations using SYCL, and oneMKL Discrete Fourier Transform (DFT) functions. This samples requires oneMKL 2024.1 (or newer).
 
 For more information on oneMKL, and complete documentation of all oneMKL routines, see https://software.intel.com/content/www/us/en/develop/tools/oneapi/components/onemkl.html.
 
-| Optimized for       | Description
-|:---                 |:---
-| OS                  | Linux* Ubuntu* 18.04; Windows 10*
-| Hardware            | Intel&reg; Skylake with Gen9 or newer
-| Software            | Intel&reg; oneMKL, Intel&reg; oneDPL
-| What you will learn | How to implement the Fourier correlation algorithm using SYCL, oneMKL, and oneDPL functions
-| Time to complete    | 15 minutes
+For more information on supported systems and the corresponding requirements, see https://www.intel.com/content/www/us/en/developer/articles/system-requirements/intel-oneapi-base-toolkit-system-requirements.html.
```

Comment:
@tomlenth, @jimmytwei could you please review README?

Reply:
Note regarding the README: I used explicit unicode characters in that file because some math symbols (_e.g._ ${&#8477;}$) would not appear as intended in the rendered version otherwise. Please give the [rendered version](https://github.com/oneapi-src/oneAPI-samples/blob/42e3c10575564edbd400c97683ebe1e4fdc31759/Libraries/oneMKL/fourier_correlation/README.md) a look and take that into consideration as you review that file.